RagnarockEditor.csproj

```diff
@@ -86,6 +86,7 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="DiscordRichPresence" Version="1.1.3.18" />
+    <PackageReference Include="Extended.Wpf.Toolkit" Version="4.6.1" />
```

Comment:
I've used this package for ColorPicker control, but now I'm reading their license and I'm not sure if this project qualifies to use the Non-Commercial License - https://github.com/xceedsoftware/wpftoolkit?tab=License-1-ov-file
@PKBeam what do you think? If it looks sketchy to you, I'll look for another open-source Color Picker control.